spec/lib/user_html_spec.rb

```diff
@@ -23,7 +23,7 @@
 
   it 'removes script tags' do
     content = 'Have a cup of <script>1337 $cr1pt</script>.'
-    expect(described_class.link_and_sanitize(content)).to eq 'Have a cup of 1337 $cr1pt.'
+    expect(described_class.link_and_sanitize(content)).to eq 'Have a cup of .'
```

Comment:
@reedstrm what caused the change in this expectation?

Reply:
Just a guess, but I bet the underlying santizer changed from "just remove the tags for script tags" to "remove the whole attempt to hack" since there's very little use-case for unwrapping the contents of a script.

Reply:
Damn you @jpslav I _just had_ to dig that up, since I was trying to ignore it, but you poked at it:
https://github.com/rgrove/sanitize/commit/faf9a0f432fda3cef29f0f8aad99d4dedf079d67

So yes, (though they _didn't_ change the docs to match! Very sloppy)

Reply:
Haha.  Thanks @reedstrm!